app/models/concerns/galtersufia/generic_file/invenio_resource_type_mappings.rb

```diff
@@ -25,7 +25,7 @@ module InvenioResourceTypeMappings
         "Collected Correspondence": ["archival", "archival-collection"],
         "Conference Proceeding": ["conference", "conference-conference_proceeding"],
         "Congresses": ["conference", "conference-congress"],
-        "Dataset": ["dataset"],
+        "Dataset": ["dataset", "dataset"],
```

Comment:
This was the only mapping that did not have two values. Adding the extra value makes the logic a little simpler in the converter. 

I did not remove the first value, the parent type, because I figure we might want it for the future. I didn't consider this before now, but since it's stored in git we could remove it without it being a problem and have only the type id stored here. If that's something we would prefer I can quickly make the change.

Reply:
Great change! Much better to add this and remove the logic/branch below.

Reply:
Thanks! Yeah, as we've gone back to update things I've found myself seriously wondering what was running through my head the first time lol.